plugins/services/src/js/containers/tasks/TasksView.js

```diff
@@ -141,25 +142,50 @@ class TasksView extends mixin(SaveStateMixin) {
     );
   }
 
-  getKillButtons(hasCheckedTasks) {
-    if (!hasCheckedTasks) {
+  getKillButtons() {
```

Comment:
Let's rename this as well as the `handleKillClick` methods to match the new wording...